Source/DotNET/Applications/Identity/ClientPrincipal.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Aksio Insurtech. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Aksio.Applications.Identity;
+
+/// <summary>
+/// Represents the structure of a Microsoft Client Principal.
+/// </summary>
+public class ClientPrincipal
+{
+#pragma warning disable CA1707, CS8618
```

Comment:
Can we add the warnings that are being disabled as a comment?

I have no idea what CA1707 and CS8618 are.

Reply:
👍🏻